ha/core/controllers/pcs/node_controller.py

```diff
@@ -114,7 +119,31 @@ def start(self, nodeid: str) -> dict:
             ([dict]): Return dictionary. {"status": "", "msg":""}
                 status: Succeeded, Failed, InProgress
         """
-        raise HAUnimplemented("This operation is not implemented.")
+        _res = self.nodes_status([nodeid])
+        _node_status = _res.get(nodeid)
+        if _node_status.lower() == NODE_STATUSES.ONLINE.value.lower():
+            return {"status": const.STATUSES.SUCCEEDED.value, "msg": f"Node {nodeid}, is already in Online status"}
+        elif _node_status.lower() == NODE_STATUSES.STANDBY_WITH_RESOURCES_RUNNING.value.lower():
+            return {"status": const.STATUSES.SUCCEEDED.value, "msg": f"Node {nodeid}, is going in standby mode, "
+                                                  f"We need to wait to complete the resource shutdown"}
```

Comment:
Don't we need to retry here again for start operation ? Or will the separate start command issued again?

Reply:
For standby node on which we do have some resources running means that node is just sent to standby mode, user need to start it again after some time.